save-plugins/fix-and-warn-plugin/src/commonMain/kotlin/org/cqfn/save/plugins/fixandwarn/FixAndWarnPlugin.kt

```diff
@@ -31,20 +31,22 @@ class FixAndWarnPlugin(
     testFiles,
     fileSystem,
     useInternalRedirections) {
-    private lateinit var fixPluginConfig: FixPluginConfig
-    private lateinit var warnPluginConfig: WarnPluginConfig
+    private val fixPluginConfig: FixPluginConfig =
```

Comment:
I made these fields `lateinit` in aim to reinitialize them after `validation`
How it can work now?

Reply:
No, you do not do that for fixPluginConfig. They are immutable, I have simply changed it to `val` and haven't broken compilation